src/components/Error/index.js

```diff
@@ -19,8 +20,16 @@ const Error = ({className, error, i18n, smaller, renderMessage = identity, rende
       <p className={style.instruction}>
         {renderInstruction(errorText.instruction)}
       </p>
+      {
+        renderAction ?
```

Comment:
You could do `{ renderAction && <div className={theme.actions}>{renderAction()}</div>}`

Reply:
Not a big fan of short-circuiting! But I can change it ;)